texdoc.cnf

```diff
@@ -369,18 +369,18 @@ alias amsproc-dev = amsclass
 alias amsmath-dev = amsmath
 
 # koma-script
-alias koma = scrguien
-alias koma-script = scrguien
-alias scrartcl = scrguien
-alias scrbook = scrguien
-alias scrreprt = scrguien
-alias typearea = scrguien
-alias koma-de = scrguide
-alias koma-script-de = scrguide
-alias scrartcl-de = scrguide
-alias scrbook-de = scrguide
-alias scrreprt-de = scrguide
-alias typearea-de = scrguide
+alias koma = scrguide-en
+alias koma-script = scrguide-en
+alias scrartcl = scrguide-en
+alias scrbook = scrguide-en
+alias scrreprt = scrguide-en
+alias typearea = scrguide-en
```

Comment:
Perhaps the suffix `-en` can be dropped here so that `texdoc` can use user's language setting?

Reply:
Without the `-en`, `texdoc koma` opens `scrguide-de.pdf` on my system, with `LC_ALL=en_CA` and `LANG=en_CA.UTF-8`. I don't really understand how aliases work though, so there may perhaps be another way.

Reply:
I think with `-en` is just fine for this case. We may want to show English version first for all locales except `de`.